src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionInitializer.java

```diff
@@ -130,34 +130,27 @@ public AlleleFractionInitializer(final AlleleFractionData data) {
     }
 
     private double estimateOutlierProbability(final AlleleFractionData data) {
-        final UnivariateObjectiveFunction objective = new UnivariateObjectiveFunction(outlierProbability ->
-                AlleleFractionLikelihoods.logLikelihood(state.shallowCopyWithProposedOutlierProbability(outlierProbability), data));
-        final SearchInterval searchInterval = new SearchInterval(0.0, MAX_REASONABLE_OUTLIER_PROBABILITY, state.outlierProbability());
-        return OPTIMIZER.optimize(objective, GoalType.MAXIMIZE, searchInterval, BRENT_MAX_EVAL).getPoint();
+        final Function<Double, Double> objective = outlierProbability ->
```

Comment:
Actually, it looks like you make calls like`AlleleFractionLikelihoods.logLikelihood(state.getParameters().copyWithNewBiasVariance(outlierProbability), state.minorFractions(), data)` below, so I think it makes sense to keep the field as an `AlleleFractionState` and make similar calls here.